docs/included-checks.md

```diff
@@ -0,0 +1,58 @@
+# ESH Guidelines Covered

```

Comment:
General comment here - can you make the check display format in each section tabular? The first column can have the name of the check (with the link to the specific page as it is now) and the second column - the severity of the check. By adding a specific color for the specific types of severity you will make a better overview of all the checks.